bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-vpn.html

```diff
@@ -0,0 +1,214 @@
+      sources=[
+        {
+          "media": "(prefers-color-scheme: light)",
+          "srcset": {
+            "img/firefox/whatsnew/whatsnew133-vpn/wnp-133-vpn-header-light.svg": "default",
+          }
+        },
+        {
+          "media": "(prefers-color-scheme: dark)",
+          "srcset": {
+            "img/firefox/whatsnew/whatsnew133-vpn/wnp-133-vpn-header-dark.svg": "default",
+          }
+        }
+      ],
+      optional_attributes={
+        "class": "wnp-main-image",
+        "alt": "Firefox",
+        "width": "450",
+        "height": "294",
+      }
+    ) }}
+    <h1 class="wnp-main-title">{{ main_title }}</h1>
+    <p class="wnp-main-tagline">{{ main_tagline }}</p>
+    <a class="mzp-c-button mzp-t-product mzp-t-xl" href="{{ url('products.vpn.landing') }}" data-cta-text="Get Mozilla VPN Now" data-cta-name="wnp-fx133-vpn-{{LANG}}-get-mozilla-vpn">
+      {{ main_cta }}
+    </a>
+  </div>
+
+  <div class="mzp-l-content mzp-t-content-lg">
+    <ul class="mzp-l-columns mzp-t-columns-two">
+      {% call picto(
+        base_el='li',
+        title=picto_1_heading,
+        image=picture(
+          url="img/firefox/whatsnew/whatsnew133-vpn/wnp-133-vpn-fast-light.svg",
+          sources=[
+            {
+              "media": "(prefers-color-scheme: light)",
+              "srcset": {
+                "img/firefox/whatsnew/whatsnew133-vpn/wnp-133-vpn-fast-light.svg": "default",
+              }
+            },
+            {
+              "media": "(prefers-color-scheme: dark)",
+              "srcset": {
+                "img/firefox/whatsnew/whatsnew133-vpn/wnp-133-vpn-fast-dark.svg": "default",
+              }
+            }
+          ],
+          optional_attributes={
+            "class": "mzp-c-picto-image",
+            "loading": "lazy",
+            "width": "292",
+            "height": "138",
+          }
+        ),
+        body=True,
+      ) %}
+        <p>{{ picto_1_desc }}</p>
+      {% endcall %}
+
+      {% call picto(
+        base_el='li',
+        title=picto_2_heading,
+        image=picture(
+          url="img/firefox/whatsnew/whatsnew133-vpn/wnp-133-vpn-safe-light.svg",
+          sources=[
+            {
+              "media": "(prefers-color-scheme: light)",
+              "srcset": {
+                "img/firefox/whatsnew/whatsnew133-vpn/wnp-133-vpn-safe-light.svg": "default",
+              }
+            },
+            {
+              "media": "(prefers-color-scheme: dark)",
+              "srcset": {
+                "img/firefox/whatsnew/whatsnew133-vpn/wnp-133-vpn-safe-dark.svg": "default",
+              }
+            }
+          ],
+          optional_attributes={
+            "class": "mzp-c-picto-image",
+            "loading": "lazy",
+            "width": "225",
+            "height": "177",
+          }
+        ),
+        body=True,
+      ) %}
+        <p>{{ picto_2_desc }}</p>
+      {% endcall %}
+    </ul>
+  </div>
+</section>
+
+<section class="wnp-footer">
```

Comment:
**nit, non-blocking**: might be worth including inline padding to match the sections above and below `--h-grid-xs` or a `mzp` container?

<img width="473" alt="Screenshot 2024-11-18 at 11 18 53 AM" src="https://github.com/user-attachments/assets/211267e4-6184-4fea-92f3-e0ac1ba35fb1">